app/models/comment.rb

```diff
@@ -56,11 +56,11 @@ def text= text
   end
 
   def people_allowed_to_be_mentioned
-    @people_allowed_to_be_mentioned ||= if parent.public?
-                                          "all"
-                                        else
-                                          [*parent.participants, parent.author].map(&:id).uniq
-                                        end
+    if parent.public?
+      "all"
+    else
+      [*parent.participants, parent.author].map(&:id).uniq
```

Comment:
you can do this with `[*parent.participations.pluck(:author_id), parent.author_id].uniq`, without joining the people table.